ifrs17/Import/Importers.ipynb

```diff
@@ -487,16 +509,13 @@
             "source": [
                 "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs primaryArgs)",
                 "\n{",
-                "\n    if (primaryArgs.Scenario != null)",
-                "\n        return primaryArgs.RepeatOnce().ToArray();",
+                "\n    if (primaryArgs.Scenario != null) return primaryArgs.RepeatOnce().ToArray();",
```

Comment:
This is a style change. I personally prefer going to new line and I always do that. 
Let's align on this.

Reply:
done